phpBB/styles/prosilver/template/mcp_post.html

```diff
@@ -14,7 +14,7 @@ <h2>{L_REPORT_DETAILS}</h2>
 			<h3>{L_REPORT_REASON}{L_COLON} {REPORT_REASON_TITLE}</h3>
 			<p class="author">{L_REPORTED} {L_POST_BY_AUTHOR} {REPORTER_FULL} &laquo; {REPORT_DATE}</p>
 		<!-- IF S_REPORT_CLOSED -->
-			<p class="post-notice reported">{{ Icon('iconify', 'fa:exclamation', '', true,  'c-topic-reported-icon') }}{L_REPORT_CLOSED}</p>
+			<p class="post-notice reported">{{ Icon('font', 'exclamation', '', true,  'fas c-topic-reported-icon') }}{L_REPORT_CLOSED}</p>
```

Comment:
fas is not `fa-solid` in font awesome 6?

Reply:
Acording to this it changes, but I am not sure https://fontawesome.com/docs/web/setup/upgrade/upgrade-from-v4#new-icon-styles

Reply:
fas is the same as fa-solid.

Reply:
Can you remove fas from here?

Reply:
The `fas` needs to be here since we specify custom classes. Otherwise fas wouldn't be included.